pvacseq/lib/main.py

```diff
@@ -216,8 +230,19 @@ def main(args_input = sys.argv[1:]):
                         help="Number of fasta entries per IEDB request. For some resource-intensive prediction algorithms like Pickpocket and NetMHC it might be helpful to reduce this number. Needs to be an even number.",
                         default=200)
     parser.add_argument("-k", "--keep-tmp-files",
-                        action='store_true', default=False,
+                        action='store_true',
                         help="Keep intermediate output files.",)
+    parser.add_argument('--net-chop',
```

Comment:
I wonder if it would make sense to not implement the `net-chop` parameter and instead have no default for `net-chop-method`. That way we save one parameter and when the `net-chop-method` parameter is set it also indicates that we want to do this processing.

We're just kinda getting up there with the number of parameters we offer so I think we should try to reduce wherever possible.